### PR TITLE
Containers

### PR DESCRIPTION
**Pull request by mkumykov**
Added a library call and example code for downloading scans that belong to a project.
Example will download all scans that belong to a project and store them in a specified folder.
API call returns a list of objects(bdio) and paths where they are stored.

### CODE REVIEW

File: blackduck/HubRestApi.py [hunk 1]
```diff
@@ -1197,6 +1197,36 @@ def remove_version_as_component(self, main_project_release, sub_project_release)
...
+            links = item['_meta']['links']
+            matches = [x for x in links if x['rel'] == 'enclosure']
+            for m in matches:
+                url = m['href']
+                filename = url.split('/')[6]
+                if output_folder:
+                    pathname = os.path.join(output_folder, filename)
+                else:
+                    pathname = filename
+                import requests
```
Comment (gsnyder2007) [inline blackduck/HubRestApi.py]: redundant?

**Comment by mkumykov** [comment]
Nope, multiple code locations lists packed in lists,

File: blackduck/HubRestApi.py (hunk 1 above)
Comment (mkumykov) [reply blackduck/HubRestApi.py]: there is not import requests at the head of the file, so I put it inline, close with the meat and away from the boilerplate code.

**Comment by gsnyder2007** [comment]
From the top of HubRestApi.py

import logging
import requests
import json

**Comment by mkumykov** [comment]
Yes, totally redundant. I took it out

**Pull request merged**